Dockerfile

```diff
@@ -59,7 +59,7 @@ ENV \
     NPM_VERSION=8.5.0 \
     PYTHON_PIPENV_VERSION=2018.11.26 \
     PYTHON_POETRY_VERSION=1.1.13 \
-    PYTHON_VIRTUALENV_VERSION=15.1.0 \
+    PYTHON_VIRTUALENV_VERSION=20.0.26 \
```

Comment:
Commit message nits:
- "python" -> "Python"
- "venv" -> "virtualenv" ([venv and virtualenv are different things](https://stackoverflow.com/a/44091914/1127485).)
- "poetry" -> "Poetry"
- "This will fix" -> "This fixes"
- "docker build" -> either "a Docker build" or "`docker build`"

Reply:
Commit message modified.